js/services/MomentAPI.js

```diff
@@ -40,12 +40,17 @@ define(function (require, exports) {
     return td.diff(fd, 'days');
   }
 
+  function formatDateToString(value) {
+    return value instanceof Date ? moment(value).toISOString(true).slice(0,10) : value;
```

Comment:
lets use format() function here. and add constant ISO_DATE_FORMAT = 'YYYY-MM-DD' for this purpose